waisntechtools/waisntechtools/settings/common.py

```diff
@@ -118,6 +121,10 @@ def environ_var(key):
     'profile'
 ]
 
+TWILIO_ACCOUNT_SID = os.environ['TWILIO_ACCOUNT_SID']
```

Comment:
can you use the `environ_var` helper function? I believe this will actually break `python manage.py test`.